### ISSUE TITLE
[DataCap Application] <Wuhan Nalai> - <Nalai Film>

### ISSUE THREAD

**Issue by bonpreu** [VenusOfficial/Pathway-VFDA#40]
### Data Owner Name

Wuhan Nalai Film and Television Culture Media Co., Ltd.

### Data Owner Country/Region

China

### Data Owner Industry

Arts & Recreation

### Website

http://www.whnlys.com/

### Social Media Handle

http://www.whnlys.com/

### Social Media Type

WeChat

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

2.4PiB

### Expected size of single dataset (one copy)

600TiB

### Number of replicas to store

4

### Weekly allocation of DataCap requested

600TiB

### On-chain address for first allocation

f3wxtso5eumhe5qubc6siqg275xnbjq7ltrcsj7wrg7aw7yb5wb2r2y47hrvwf4hqn3xjk7kjyhj2jgrhbq4ma

### Data Type of Application

Private Commercial/Enterprise

### Custom multisig

- [X] Use Custom Multisig

### Identifier

 Use Custom Multisig

### Share a brief history of your project and organization

```text
Wuhan Nalai is a new media film and television production company. It's main business includes production and joint production, publicity and distribution, radio and television program production, performance brokerage, etc. It is committed to disseminating excellent traditional culture and creating high-quality new media film and television.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### If answered yes, what are the other projects/ecosystem stakeholders

```text
No response
```

### Describe the data being stored onto Filecoin

```text
Trailers, samples, photos, recordings of the films produced by our company.
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you answered "Other" in the previous question, enter the details here

```text
Copyrighted movie data, currently stored in IDC.
```

### If you are a data preparer. What is your location (Country/Region)

China

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
Graphsplit,I prefer offline transmission
```

### If you are not preparing the data, who will prepare the data?  (Provide name and business)

```text
regional data will send to sp they will process
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
No response
```

### Please share a sample of the data

```text
http://www.whnlys.com/vod_show.asp?id=179
http://www.whnlys.com/vod_show.asp?id=177
http://www.whnlys.com/vod_show.asp?id=93
http://www.whnlys.com/ysxm_show.asp?id=102
http://www.whnlys.com/vod_show.asp?id=94
http://www.whnlys.com/news_show.asp?id=349
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [ ] I confirm

### If you chose not to confirm, what was the reason

```text
This is private data for archival and internal employee use only. For E-Fil+ project.
```

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

More than 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China

### How will you be distributing your data to storage providers

Others

### How did you find your storage providers

Slack, Partners

### If you answered "Others" in the previous question, what is the tool or platform you used

```text
No response
```

### Please list the provider IDs and location of the storage providers you will be working with.

```text

1, f02001818 Hong Kong
2, f02002218 Seoul, South Korea
3, f02002233 Singapore
4, f02003000 Changsha, China

```

### How do you plan to make deals to your storage providers

Boost client, Droplet client

### If you answered "Others/custom tool" in the previous question, enter the details here

```text
No response
```

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by bonpreu** [comment]
@VenusOfficial If possible, we have more space, can you give us the total required 2 PiB, thank you very much.

**Comment by bonpreu** [comment]
@VenusOfficial 
We will add a new node: f03173127
Location: Dallas, USA, The KYC certificate has been sent to your email, thank you!

**Comment by VenusOfficial** [comment]
@bonpreu, I'm pleased to have received your new DC application. The KYB and some SP's KYC documents from your previous application remain valid. The new SP f03173127 has successfully passed verification based on the details provided in your email. I can allocate 1.2 PiB to you before the allocator reaches the DC's capacity limit.

**Comment by VenusOfficial** [comment]
@bonpreu The compliance report indicates that some of your data's CIDs match those of other clients. Could you please provide an explanation for this?

**Comment by bonpreu** [comment]
@VenusOfficial
 I apologize for any inconvenience. The issue highlighted in the report may resulte from the data we have been sharing with others. We will stop sharing data from now on.

**Comment by bonpreu** [comment]
@VenusOfficial
We will add a new node: f03137648
Location: New York, USA, thank you!

**Comment by bonpreu** [comment]
@VenusOfficial 
Hello, when will the remaining 100T be allocated? In addition, I also want to apply for the 1P quota. Can I continue to replenish it? Thanks

**Comment by VenusOfficial** [comment]
@bonpreu, Only 300t is available for the allocator. I'll assign it to you if that works for you.

**Comment by bonpreu** [comment]
@VenusOfficial
It's fine to me.

**Comment by bonpreu** [comment]
@VenusOfficial 
Hello, I have updated my application information. I still need 1.2PB of data storage, plus the 1.2PB you provided me, a total of 2.4PB, these two new SPs:
f0867300 Tokyo, Japan
f01993339 Singapore
We have communicated and support retrieval. I will provide you with the relevant KYC information via email later. I hope to get your approval as soon as possible. Thank you!

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
@bonpreu 
I've filtered the data to include only the SPs participating in this round of DC assignment. Could you clarify why the deal percentages vary significantly among these SPs and why their retrieval rates are all blank?

<img width="716" alt="image" src="https://github.com/user-attachments/assets/38ac6274-dbee-49b6-b6e5-58fe4e1fd443">

**Comment by bonpreu** [comment]
@VenusOfficial 
These SPs were just launched yesterday. We made 4 copies of one piece of data and distributed them evenly, so the completion percentage is relatively even. The retrieval rate is because the robot has not yet come to search. We support retrieval. Please help us continue with the approval. Thank you!

**Comment by VenusOfficial** [comment]
@bonpreu 
The DC has been assigned for over a week, but little progress has been made. Could you expedite the process?

**Comment by bonpreu** [comment]
@VenusOfficial 
We are very sorry for the inconvenience caused to you. Due to the recent high gas fees, we have suspended our services. We have now resumed some services. Thank you!

**Comment by bonpreu** [comment]
@VenusOfficial 
We have been communicating with the SP. Due to the recent high gas fees, the SP that submitted it previously decided not to encapsulate it anymore. Now we have found a new SP to cooperate with. I will update the application and send the KYC information to your email. I am very sorry for the inconvenience caused to you.

**Comment by VenusOfficial** [comment]
@bonpreu 
Thank you for your explanation. I understand that the high base gas fee poses challenges for SP operations. The locations of the new SPs mentioned in your email have been verified, and KYC has been completed. The previously assigned DC will remain unchanged. However, I will monitor the report more frequently moving forward. Please ensure full compliance. Thank you, as always, for your cooperation.
<img width="403" alt="image" src="https://github.com/user-attachments/assets/3d992a50-2e9c-4737-8c16-2e259a11c285" />

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
The zero retrieval rate for the 4 SPs might be due to a delay in data synchronization, as the sealed data is currently very small. I will run the report again later to verify the retrieval status.
<img width="758" alt="image" src="https://github.com/user-attachments/assets/75d642e2-8148-4e8c-b2ca-935d788ae21d" />

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
@bonpreu 

The report shows that the retrieval rate for the 4 SPs is 0. Could you please check this?

**Comment by bonpreu** [comment]
@VenusOfficial 
Yes, our technicians have found some errors and are currently working on resolving them. Please be patient and we will contact you once we have resolved them. In addition, we still have a storage demand of 4P. Can you continue to provide it?

**Comment by VenusOfficial** [comment]
@bonpreu 
We have run out of DC and plan to apply for a refresh shortly after receiving your retrieval rate report. Your additional request for new DC will be considered in our next round of allocation. Therefore, please resolve your retrieval issue as soon as possible. Thank you.

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by Filedrive-ops** [comment]
@bonpreu 
Compared to yesterday's report, the retrieval rate has significantly improved today, increasing from nearly 0 to approximately 10% across the 4 SPs. Great job! Thank you in advance for your continued efforts to further tuning. 
<img width="645" alt="image" src="https://github.com/user-attachments/assets/1141614b-ccab-4971-ac3b-594f584c1bd0" />

**Comment by VenusOfficial** [comment]
@bonpreu
Compared to yesterday's report, the retrieval rate has significantly improved today, increasing from nearly 0 to approximately 10% across the 4 SPs. Great job! Thank you in advance for your continued efforts to further tuning.
<img width="645" alt="图片" src="https://github.com/user-attachments/assets/2ba37852-117e-4fe9-a05a-5bbef0c9e201" />

**Comment by bonpreu** [comment]
@VenusOfficial 
Yes, we would also like to thank the technical staff of the VENUS team for cooperating with us to locate the problem. We have now found the problem and will continue to improve the search. Thank you again for your cooperation!

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
@bonpreu 
Thank you for your efforts in improving the average retrieval rate from 10% to approximately 40%.
<img width="688" alt="图片" src="https://github.com/user-attachments/assets/f338a97d-73e5-4afc-9f55-3b8d8cee1e6a" />

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
Retrieval rate was improved from 37% to 45%.
<img width="702" alt="图片" src="https://github.com/user-attachments/assets/0c9d1ad2-06c4-4ee6-b28f-8c73f7b1c498" />

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
Retrieval rate was improved from 45% to 47%.
<img width="702" alt="图片" src="https://github.com/user-attachments/assets/c5e86154-13c9-437d-8e48-3d2cd01e1769" />

**Comment by VenusOfficial** [comment]
checker:manualTrigger

**Comment by VenusOfficial** [comment]
@bonpreu 

Could you check why the percentage of a single copy of data is as high as 57%?

**Comment by bonpreu** [comment]
@VenusOfficial 
 After reviewing the log file of the deal using the following command, it was found that the number of 4 copies is 7,403, accounting for more than 77% of the total copies. The issue mentioned in the report may be related to delays in retrieving data from the chain.
cat /tmp/f0*.log | awk '{print $4}' | sort -n | uniq -c | sort -n -k1 | grep '4 ba' | wc -l

**Comment by VenusOfficial** [comment]
@bonpreu 

Since all your DC requests have been fulfilled, I am now closing your application. Please let me know if you have any questions or concerns.
